supervisor/shared/web_workflow/web_workflow.c

```diff
@@ -428,7 +428,6 @@ static bool _endswith(const char *str, const char *suffix) {
 }
 
 const char *ok_hosts[] = {
-    "code.circuitpython.org",
```

Comment:
This was leftover from when we thought we could get browsers to let us cross domain request to local. However, they block it. So, this isn't needed.